src/kernel/diffusive/diffusive.f90

```diff
@@ -892,6 +944,17 @@ subroutine diffnw(timestep_ar_g, nts_ql_g, nts_ub_g, nts_db_g, ntss_ev_g, nts_qt
           deallocate(used_lfrac)
           deallocate(flag_lfrac)   
         endif
+        
+            !* test
+            !do ts=1, ntss_ev_g
+            !do jm = 1, nmstem_rch
+            !    j = mstem_frj(jm)
+            !do i=1, frnw_g(j,1)
+            !    write(101,"(f10.1, 2I10, 4F20.4)") saveInterval*real(ts-1)/60.0, i, j, q_ev_g(ts, i, j),&
```

Comment:
There is a lot of commented out/test/diagnostic code in this function that should probably get removed